apps/i18n/common/en_us.json

```diff
@@ -1161,6 +1161,8 @@
   "schoolCity": "School City",
   "schoolCityTown": "City / Town",
   "schoolCountry": "School Country",
+  "schoolInfoDialogDescription": "Welcome back!  Are you still teaching at",
```

Comment:
We'll probably want to make it easier to pass a parameter to this string, like: https://github.com/code-dot-org/code-dot-org/blob/2e9ce00cc2e5c3368efccb045705f2de8e20fa9d/apps/i18n/common/en_us.json#L72

Reply:
@epeach Good point, this is implemented in the follow-up PR